resources/lang/id/authorization.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'beatmap_discussion' => [
+        'destroy' => [
+            'is_hype' => 'Can not undo hyping.',
+            'has_reply' => 'Can not delete discussion with replies',
+        ],
+        'nominate' => [
+            'exhausted' => 'You have reached your nomination limit for the day, please try again tomorrow.',
+        ],
+        'resolve' => [
+            'not_owner' => 'Only thread starter and beatmap owner can resolve a discussion.',
+        ],
+
+        'vote' => [
+            'limit_exceeded' => 'Please wait a while before casting more votes',
+            'owner' => "Can't vote on own discussion.",
+            'wrong_beatmapset_state' => 'Can only vote on discussions of pending beatmaps.',
+        ],
+    ],
+
+    'beatmap_discussion_post' => [
+        'edit' => [
+            'system_generated' => 'Automatically generated post can not be edited.',
+            'not_owner' => 'Only the poster can edit post.',
+        ],
+    ],
+
+    'chat' => [
+        'channel' => [
+            'read' => [
+                'no_access' => 'Access to requested channel is not permitted.',
```

Comment:
'no_access' => 'Akses ke saluran yang diminta tidak diizinkan.',

Reply:
I'll change saluran to kanal for consistency sake.